mpqp/core/instruction/gates/native_gates.py

```diff
@@ -587,8 +599,6 @@ def __init__(self, a: int, b: int):
 
         self.qlm_aqasm_keyword = "SWAP"
 
-        self.matrix = np.array([[1, 0, 0, 0], [0, 0, 1, 0], [0, 1, 0, 0], [0, 0, 0, 1]])
```

Comment:
is it a mistake ?

Reply:
I re-added it, explain me if it was a mistake